docs/topics/multiplatform-mobile/multiplatform-mobile-wrap-up.md

```diff
@@ -47,7 +47,7 @@ additional cross-platform mobile development tasks:
      <ul>
         <li><a href="multiplatform-mobile-integrate-in-existing-app.md">Make your Android app cross-platform</a></li>
         <li><a href="https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/">Kotlin Multiplatform Hands-on: Networking and Data Storage</a></li>
-        <li><a href="https://play.kotlinlang.org/hands-on/Full%20Stack%20Web%20App%20with%20Kotlin%20Multiplatform/">Building a Full Stack Web App with Kotlin Multiplatform</a></li>
+        <li><a href="multiplatform-full-stack-app.md">Building a Full Stack Web App with Kotlin Multiplatform</a></li>
```

Comment:
JFYI
this hand-on tutorial was moved to `multiplatform-full-stack-app.md`
looks like all the other links were fixed, but this one still not